src/bionics.cpp

```diff
@@ -1543,15 +1548,33 @@ void player::introduce_into_anesthesia( const time_duration &duration, player &i
                            _( "You settle into position, sliding your right wrist into the couch's strap." ),
                            _( "<npcname> settles into position, sliding their wrist into the couch's strap." ) );
     if( anesthetic ) {
+        if( has_trait( trait_THRESH_MEDICAL ) ) {
+            add_msg_if_player( m_mixed,
+                               _( "You feel excited, as operation starts." ) );
+        }
+
         add_msg_if_player( m_mixed,
                            _( "You feel a tiny pricking sensation in your right arm, and lose all sensation before abruptly blacking out." ) );
 
         //post-threshold medical mutants with Deadened don't need anesthesia due to their inability to feel pain
     } else {
+        //post-threshold medical mutants do not fear operations.
+        if( has_trait( trait_THRESH_MEDICAL ) ) {
+            add_msg_if_player( m_mixed,
+                               _( "You feel excited, as the Autodoc slices painlessly into you.  You pass out when the blades reach your line of sight." ) );
```

Comment:
Don't fears of pain because of Deadened, excited because of Prototype, but still passes out? Why? I guess we need an explanation for this.

Reply:
Operation is stress for body. So let's go for "suddenly feel tired".